docs/register_solution_group.md

```diff
@@ -35,6 +35,7 @@ async function main(): Promise<void> {
   }
   const operation_start_block = 10
   const operation_end_block = 1000
+  const withdrawal_delay = 5
 
   // Registering of solution group reserves part of the free balance. The amount of the reserved funds can be queried as `registrarDeposit()`
```

Comment:
should we add a quick notice on the withdrawal_delay param ? (I am even wondering if we should not right our comments in docstring-ish way
```suggestion
  // Registering of solution group reserves part of the free balance. The amount of the reserved funds can be queried as `registrarDeposit()`
   // @dev : the withdrawal delay is the number of blocks to wait before withdrawal request is executed
```